src/modules/network.json

```diff
@@ -0,0 +1,467 @@
+        "summary": "Successful Response after connecting to the Wifi.",
+        "schema": {
+          "$ref": "#/components/schema/NetworkStatus"
+        }
+      },
+      "examples": [
+        {
+          "name": "Connected to a Wifi Router",
+          "params": [],
+          "result": {
+            "name": "Default Result",
+            "value": {
+              "connection": true,
+              "interface": "Wifi",
+              "internet": true
+            }
+          }
+        },
+        {
+          "name": "Connected to Ethernet",
+          "params": [],
+          "result": {
+            "name": "Default Result",
+            "value": {
+              "connection": true,
+              "interface": "Ethernet",
+              "internet": true
+            }
+          }
+        },
+        {
+          "name": "Connected to Wifi but no internet",
+          "params": [],
+          "result": {
+            "name": "Default Result",
+            "value": {
+              "connection": true,
+              "interface": "Ethernet",
+              "internet": false
+            }
+          }
+        },
+        {
+          "name": "Not connected",
+          "params": [],
+          "result": {
+            "name": "Default Result",
+            "value": {
+              "connection": false,
+              "interface": "None",
+              "internet": false
+            }
+          }
+        }
+      ]
+    },
+    {
+      "name": "onConnectionChanged",
+      "summary": "Registers as a provider for when the Network or Internet connection fluctuates.",
+      "params": [
+        {
+          "name": "listen",
+          "summary": "Whether or not to listen for requests to this interface method.",
+          "schema": {
+            "type": "boolean"
+          }
+        }
+      ],
+      "result": {
+        "name": "status",
+        "summary": "The network status change event",
+        "schema": {
+          "oneOf": [
+            {
+              "$ref": "https://meta.comcast.com/firebolt/types#/definitions/ListenResponse"
+            },
+            {
+              "$ref": "#/components/schema/NetworkStatus"
+            }
+          ]
+        }
+      },
+      "examples": [
+        {
+          "name": "Default Example",
+          "params": [
+            {
+              "name": "listen",
+              "value": true
+            }
+          ],
+          "result": {
+            "name": "Default Result",
+            "value": {
+              "connection": true,
+              "interface": "Wifi",
+              "internet": true
+            }
+          }
+        }
+      ]
+    },
+    {
+      "name": "wps",
+      "summary": "Connect to WPS",
+      "params": [
+        {
+          "name": "security",
+          "schema": {
+            "$ref": "#/components/schema/WPSSecurityPin"
+          }
+        }
+      ],
+      "result": {
+        "name": "Connected Wifi",
```

Comment:
result names should be safe variable names. we use them to generate the code examples in the docs